UPGRADE.md

```diff
@@ -0,0 +1,12 @@
+UPGRADE FROM 4.0 to 5.0
+=======================
+
+   - The web configurator got removed. So you need to remove the `_configurator`
```

Comment:
Can you fix CS? We are using `*` for list and only use one space to indent it. Can you also add a blank line between each item? Thanks